docker/conf/webauthn-apache2.conf

```diff
@@ -0,0 +1,31 @@
+<Virtualhost *:80>
+    ServerName  webauthn
```

Comment:
Thought for a later time: We could base the servername on a provided env_var

Reply:
Very true! Like we discussed, we might make a shared config template based on env vars. And for "hostname" specifically, it could be generic (I think it accepts "*" or something like that, need to remember and look it up).